back/spec/acceptance/projects_spec.rb

```diff
@@ -30,7 +30,8 @@
 
       parameter :areas, 'Filter by areas (AND)', required: false
       parameter :publication_statuses, 'Return only projects with the specified publication statuses (i.e. given an array of publication statuses); returns all projects by default', required: false
-      parameter :filter_can_moderate, 'Filter out the projects the user is allowed to moderate. False by default', required: false
+      parameter :filter_can_moderate, 'Filter out the projects the current_user is allowed to moderate. False by default', required: false
+      parameter :filter_user_is_moderator_of, 'Filter out the publications the given user is not moderator of. False by default', required: false
```

Comment:
Should be projects here, not publications, right?

Reply:
Yes, true!